SYNKRON-264 Implement APIs in Iconic SDK

jinalpanchal123 opened the pull request:
Implement following APIs:
- <https://api.sellercenter.com.br/docs/#/Orders/GET_v2-orders-order\\\>
- <https://api.sellercenter.com.br/docs/#/Orders/GET_v2-order-items-get\\\>
- <https://api.sellercenter.com.br/docs/#/Product/get_v2_products__productId__consignment_stocks\\\>
- <https://api.sellercenter.com.br/docs/#/Product/get_v2_product_shop_sku__shopSku_\\\>
- <https://api.sellercenter.com.br/docs/#/Product/get_v2_product_seller_skus\\\>
- <https://api.sellercenter.com.br/docs/#/Webhook/get_v2_webhook_entities\\\>

jinalpanchal123 committed:
In TheIconic-rest:
Implement following APIs:
- <https://api.sellercenter.com.br/docs/#/Orders/GET_v2-orders-order\\\>
- <https://api.sellercenter.com.br/docs/#/Orders/GET_v2-order-items-get\\\>
- <https://api.sellercenter.com.br/docs/#/Product/get_v2_products__productId__consignment_stocks\\\>
- <https://api.sellercenter.com.br/docs/#/Product/get_v2_product_shop_sku__shopSku_\\\>
- <https://api.sellercenter.com.br/docs/#/Product/get_v2_product_seller_skus\\\>
- <https://api.sellercenter.com.br/docs/#/Import/post_v2_import_stock_update\\\>
- <https://api.sellercenter.com.br/docs/#/Webhook/get_v2_webhook_entities\\\>

dftrakesh commented on src/main/java/com/dft/theiconic/OrdersApi.java:
```diff
@@ -36,6 +35,19 @@ public OrdersResponse getOrders(HashMap<String, String> parameters) {
         return getRequestWrapped(request, handler);
     }
 
+    public OrdersResponse getOrders(String orderId) {
```
Rename method name getOrders to getOrderById

jinalpanchal123 replied:
yes sir

dftrakesh commented on src/main/java/com/dft/theiconic/OrdersApi.java:
```diff
@@ -36,6 +35,19 @@ public OrdersResponse getOrders(HashMap<String, String> parameters) {
+        return getRequestWrapped(request, handler);
+    }
+
+    public OrdersResponse getOrderItems() {
```
Create new class OrderItemsResponse

jinalpanchal123 replied:
created sir

jinalpanchal123 committed:
In the iconic-rest:
- changes made as per PR comment:
  - Create new class OrderItemsResponse
  - Rename method name getOrders to getOrderById

jinalpanchal123 committed:
In the iconic-rest:
- return OrderItem instead of Order in OrderItemsResponse
